### DISCUSSION TITLE
Running WBOT on Container on Portioner?

### DISCUSSION THREAD

**Discussion by ammaralfarsi** [vasani-arpit/WBOT#364] [Q&A]
Is it possible to run WBOT on Container in Portioner? if yes then could you please explain how?

**Comment by vasani-arpit** [comment]
Yes. It is possible. 

- Option 1: You can use the binary and use ubuntu as base image. I haven't tested it this way but it should work theoretically. It will require all the dependencies which puppeteer(chrome) needs in ubuntu which is the following 
    ```sudo apt update && sudo apt install -y gconf-service libgbm-dev libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget```

- Option 2: You can take the source and dockerize it, You will need the same puppeteer dependencies and additionally you will need node LTS. 

That's it.

**Reply by ammaralfarsi** [reply]
thank you for your reply, but wasn't sure how to do that 😅

i'm running the Portainor within Home Assistant OS. so any help ?
